rfcs/QueryLevelNullability.md

```diff
@@ -100,7 +99,7 @@ type's properties would have the desired nullability.
 
 ## 🧑‍💻 Proposed Solution
 
-A client specified Non-Null designator.
```

Comment:
The capitalization looks a little weird, but it was intended to relate this to the existing SDL-side [Non-Null ](https://spec.graphql.org/June2018/#sec-Type-System.Non-Null) type

Reply:
ohhhhh I was wondering what was up with that! I'll revert it.